bindings/cs/MongoDB.Crypt.Test/BasicTests.cs

```diff
@@ -461,89 +429,89 @@ public void TestLocalKeyCreationStepwise()
             switch (context.State)
             {
                 case CryptContext.StateCode.MONGOCRYPT_CTX_NEED_MONGO_COLLINFO:
-                {
-                    var binary = context.GetOperation();
-                    var doc = BsonUtil.ToDocument(binary);
-                    _output.WriteLine("ListCollections: " + doc);
-                    var reply = ReadJsonTestFile("collection-info.json");
-                    _output.WriteLine("Reply:" + reply);
-                    context.Feed(BsonUtil.ToBytes(reply));
-                    context.MarkDone();
-                    return (CryptContext.StateCode.MONGOCRYPT_CTX_NEED_MONGO_COLLINFO, binary, doc);
-                }
+                    {
```

Comment:
Looks like a lot of whitespace changes made it in by accident?

Reply:
it's a default format from `vs` via `ctrl` + `k` + `d`. I guess it can be configured somehow or these lines can be manually reverted. Not sure that it worth it?

Reply:
Hm, if we're going to make style changes, how about we drop the braces that surround the contents of each "case" in the switch statement? That should make the autoformatter drop the spaces in front of each line.

Reply:
Discussed offline, let's just leave it, I don't mind 😆